src/preseed/ocemr_installer.cfg

```diff
@@ -1 +1,10 @@
-d-i preseed/include string common.cfg i386.cfg
+# Install on an ultra small disk 4gb
+#d-i preseed/include string common.cfg disk-4g.cfg config.cfg
+
+# Install on any normal single hard drive
+#d-i preseed/include string common.cfg disk-auto.cfg config.cfg
+
+# Install on a software raid
+d-i preseed/include string common.cfg disk-raid.cfg config.cfg
```

Comment:
![screenshot_ocemr-test_2018-08-14_21 23 52](https://user-images.githubusercontent.com/39502/44131545-9c1dd59c-a008-11e8-99e7-0e5104670168.png)

I guess you're making some assumptions about a specific hardware platform now.

Reply:
Updated README to add to the configure step.